packages/paste-core/components/meter/src/MeterLabel.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import {type BoxProps, Box} from '@twilio-paste/box';
+import {Label} from '@twilio-paste/label';
+import type {HTMLPasteProps} from '@twilio-paste/types';
+
+import {labelIdSuffix} from './Meter';
+
+export interface MeterLabelProps extends HTMLPasteProps<'div'>, Pick<BoxProps, 'element'> {
+  children: string;
+  htmlFor: string;
+}
+
+const MeterLabel = React.forwardRef<HTMLLabelElement, MeterLabelProps>(
+  ({element = 'METER_LABEL', children, htmlFor, ...labelProps}, ref) => {
+    return (
+      <Box display="flex" width="fit-content" element={`${element}_WRAPPER`}>
```

Comment:
question: I wonder if `display="flex" width="fit-content"` is equivalent to just doing `display="inline"`?

Reply:
Seems fine with `display="inline"`! These styles will likely change after design work though

Reply:
actually just got rid of the label wrapper for now